crowbar_framework/app/models/node_object.rb

```diff
@@ -835,6 +840,20 @@ def _remove_elements_from_node(old, new, from_node)
     @attrs_last_saved = @role.default_attributes.deep_dup
 
     Rails.logger.debug("Done saving node: #{@node.name} - #{crowbar_revision}")
+
+    unless @notify_on_save.empty?
+      Rails.logger.debug("Notifying about saved node: #{@node.name}")
+
+      details = {
+        node: @node.name,
+        attributes: @notify_on_save.keys
```

Comment:
As I mentioned earlier, we only send the keys, not the values. So we might want to send the values as well, or use a single set as mentioned earlier.